src/lib/mathlib/math/Vector.hpp

```diff
@@ -298,6 +298,18 @@ class __EXPORT VectorBase
 	}
 
 	/**
+	 * element by element inversion
+	 */
+	const Vector<N> einv() const {
+		Vector<N> res;
+
+		for (unsigned int i = 0; i < N; i++)
+			res.data[i] = 1 / (data[i] + FLT_EPSILON);
```

Comment:
I just got rid of the protection, in keeping with rest of the implementation in the library.